examples/cifar10/cifar10-real-world/virtualenv/min-requirements.txt

```diff
@@ -0,0 +1,5 @@
+nvflare>=2.2.1rc6
```

Comment:
might need to change this when release but now is fine

Reply:
Yes, we'll need to change again after release. I changed to rc7 for now.